doc/alarm.md

```diff
@@ -184,17 +188,30 @@ public interface SmsSender {
 
 **C. Webhook alarm service**
 
-Webhook alarm service is the api of Pinpoint alarm over HTTP. To receive alarms you will need to prepare the receiver server. There is an example of webhook slack receiver project, see 3. Others - [5) An example of webhook receiver project](##3.Others). The payloads of the webhook have the different schemas depending on the alarm checker type. you can see the example of the payload schemas in 3.Others - [6) The Specification of webhook payloads and the examples](##3.Others). If you have an previous release of pinpoint-2.1.1, you should change the schema of alarm_rule table.
+Webhook alarm service is the API of Pinpoint alarm over HTTP. 
+
+**To receive alarms, you will need to set up the receiver server, or You can use an example of webhook slack receiver project. see [An example of webhook receiver project](https://github.com/doll6777/slack-receiver).**
```

Comment:
**The webhook receiver service that receives the webhook message should be implemented by your own, or use a [sample project](https://github.com/doll6777/slack-receiver) provided** (in this case Slack)